package/MDAnalysis/analysis/base.py

```diff
@@ -198,25 +248,33 @@ def run(self, start=None, stop=None, step=None, verbose=None):
 
 
 class AnalysisFromFunction(AnalysisBase):
-    """
-    Create an analysis from a function working on AtomGroups
+    r"""Create an :class:`AnalysisBase` from a function working on AtomGroups
 
-    Attributes
-    ----------
-    results : ndarray
-        results of calculation are stored after call to ``run``
+    .. code-block:: python
 
-    Example
-    -------
-    >>> def rotation_matrix(mobile, ref):
-    >>>     return mda.analysis.align.rotation_matrix(mobile, ref)[0]
+        def rotation_matrix(mobile, ref):
+            return mda.analysis.align.rotation_matrix(mobile, ref)[0]
 
-    >>> rot = AnalysisFromFunction(rotation_matrix, trajectory, mobile, ref).run()
-    >>> print(rot.results)
+        rot = AnalysisFromFunction(rotation_matrix, trajectory,
+                                   mobile, ref).run()
+        print(rot.results.timeseries)
+
+    Attributes
+    ----------
+    results.frames : numpy.ndarray
+            simulatiom frames taken for evaluation
+    results.times : numpy.ndarray
+            simulatiom times taken for evaluation
+    results.timeseries : asarray
+        Results for each frame of the underlaying function
+        stored after call to ``run``.
 
     Raises
     ------
     ValueError : if ``function`` has the same kwargs as ``BaseAnalysis``
+
+    .. versionchanged:: 2.0.0
+        Former `results` are now stored as `results.timeseries`
```

Comment:
use markup
```suggestion
        Former :attr:`results` are now stored as :attr:`results.timeseries`
```
It won't link but be formatted consistently.